src/main/java/com/bank/Application/InMemoryDatabase.java

```diff
@@ -0,0 +1,33 @@
+package main.java.com.bank.Application;
+
+import java.util.HashMap;
+
+public class InMemoryDatabase {
+
+    private static InMemoryDatabase instance;
+
+    private HashMap<String, Object> storage;
+
+    private InMemoryDatabase(){
+        storage = new HashMap<>();
+    }
+
+    public static InMemoryDatabase getInstance(){
+        if (instance == null){
+            instance = new InMemoryDatabase();
+        }
+        return instance;
+    }
+
+    public HashMap<String, Object> getStorage() {
+        return storage;
+    }
+
+    //apenas para testes
```

Comment:
Get used to always use the same language in the whole code base, consistency is important.